archetype/maven-url-handler/src/main/java/io/helidon/build/archetype/maven/url/mvn/Connection.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.helidon.build.archetype.maven.url.handler;
+package io.helidon.build.archetype.maven.url.mvn;
```

Comment:
Why the change of package name ? `maven.url.mvn` is redudant, I'd say `maven.url` was fine.

Reply:
The `.mvn` and `Handler` were chosen for a naming convention which allows these packages/class to be used when a `URL` with `mvn` as a protocol is instantiate. 

So in the new commit I made the recommended changes and create a custom URLStreamHandlerFactory instead.